### PR TITLE
Add transEscAttr view helper and use it in templates.

### PR DESCRIPTION
**Pull request by EreMaijala**
While transEsc alone usually works fine, it doesn't hurt to do things as they should, and this shortens the translate+escapeHtmlAttr chain in templates.

TODO:
- [x] Run full test suite

### CODE REVIEW

File: themes/bootstrap3/templates/search/searchbox.phtml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<a href="FOO" title="<?=$this->transEsc('"Phoo"')?>">Pho</a>
```
Comment (demiankatz) [inline themes/bootstrap3/templates/search/searchbox.phtml]: I'm not sure what this is, but I'm guessing it's not supposed to be here. :-)

**Review by demiankatz — CHANGES_REQUESTED** [review]
This looks good apart from one bit of code that appears to have been added by accident. If time permits later, I may just remove that myself. :-) I'll also run the full test suite on this to be 100% sure everything still works before merging, but a careful code review suggests to me that it should be fine.

File: themes/bootstrap3/templates/search/searchbox.phtml (hunk 1 above)
Comment (EreMaijala) [reply themes/bootstrap3/templates/search/searchbox.phtml]: Don't you like it? :D Err.. I'll remove it.

Comment (demiankatz) [reply themes/bootstrap3/templates/search/searchbox.phtml]: Thanks! All tests are (unsurprisingly) passing, so I'll merge this soon.

**Pull request merged**